src/domain/journey/space/pages/SpaceSettings/SpaceSettingsView.tsx

```diff
@@ -284,6 +330,25 @@ export const SpaceSettingsView: FC<SpaceSettingsViewProps> = ({ journeyId, journ
               }}
             />
           </PageContentBlock>
+
+          <PageContentBlock sx={{ borderColor: theme.palette.error.main }}>
+            <PageContentBlockHeader
+              sx={{ color: theme.palette.error.main }}
+              title={t('components.deleteSpace.title')}
+            />
+            <Box display="flex" gap={1} alignItems="center" sx={{ cursor: 'pointer' }} onClick={openDialog}>
+              <DeleteIcon />
+              <Caption>{t('components.deleteSpace.description', { entity: translatedEntity })}</Caption>
+            </Box>
+          </PageContentBlock>
+          {openDeleteDialog && (
+            <SpaceProfileDeleteDialog
+              entity={translatedEntity}
```

Comment:
I think we should pass `loading` so we can disable (and show loading) inside the button during deletion. 
You can get the loading from `useDeleteSpaceMutation`. A button with loading is available in create space/subspace.